frontend/public/index.html

```diff
@@ -1,37 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <% if (process.env.NODE_ENV !== 'development') { %>
-    <meta
-        http-equiv="Content-Security-Policy"
-        content="
+
+<head>
+  <% if (process.env.NODE_ENV !=='development' ) { %>
+    <meta http-equiv="Content-Security-Policy" content="
             %REACT_APP_CONTENT_SECURITY_POLICY_DEFAULT_SRC%
             %REACT_APP_CONTENT_SECURITY_POLICY_SCRIPT_SRC%
             style-src 'self' 'unsafe-inline';
-            img-src 'self' https://hhs-prime.okta.com data:;
+            img-src 'self' https://hhs-prime.oktapreview.com data:;
```

Comment:
Just want to confirm that this is ok, regarding the hardcoding of oktapreview here. Will this affect higher environments ?

Reply:
Good eyes, but this isn't permanent - I was just testing a change.